### ISSUE TITLE
Qubino RGBW dimmer ZMNHWD1 recognized as 4 switches instead of level+ RGBW dimmer

### ISSUE THREAD

**Issue by pchevali** [domoticz/domoticz#1683]
Hi, I am running latest git master version (3.8092 built from sources ), the system is Arch Linux.

The qubino ZMNHWD1  device is a zwave RGBW led dimmer. As stated in the  following websites  ( in french but screen captures are explicit enough I think) , this component had been recognized in past as a Level switch combined with an RGBW dimmer ( Lighting limitless with color panel to pick up color) 
The websites :
https://easydomoticz.com/module-qubino-flush-rgbw-dimmer-zmnhwd1-domoticz/
http://www.planete-domotique.com/blog/2017/04/24/controleur-rgbw-de-qubino-ajoute-couleurs-a-domotique/

According to them the version they used is 3.7450, unfortunatly I didn't test with this version, but the master branch doesn't give me the same result. After device association I have 4 level switch ( dimmers) for each color.

I can't find any references to such change in the commit history.

Here are the devices : 
![image](https://user-images.githubusercontent.com/1221917/27993848-bf027a9c-64b1-11e7-8572-c16371e5f5a3.png)

And There the hardware  module 
![image](https://user-images.githubusercontent.com/1221917/27993860-df21c7b0-64b1-11e7-9f8f-746705712e41.png)

There is the openzwave version I am using ( usb dongle ): 
OpenZWave USB Version: 1.4-2642-gc42a15e9-dirty

**Comment by pchevali** [comment]
I had a look at the code and it seems that the part that handles the creation of the RGBW switch is in hardware/ZWaveBase.cpp , in the function IsNodeRGBW() 
At the end of the function there is a check on the description of the device if the string contains "FGRGBWM441" :
return (ProductDescription.find("FGRGBWM441") != std::string::npos);

As the Qubino ZMNHWD1 is similar to the Fibaro device FGRGBWM441, the 2 devices should behave the same, is adding a check for the ZMNHWD1 device in description a solution ? (Maybe ZMNHWD only because the trailing number identifies the frequency of the device )

Thanks

**Comment by gizmocuz** [comment]
Yes that will solve this.. making a patch now... small moment

**Comment by gizmocuz** [comment]
Done, should work in beta 8150 and up... (could take an hour before it is build)
Please give some feedback if it's working

**Comment by pchevali** [comment]
Hi gizmocuz,

Sorry I didn't report since then. I did  test this solution but this didn't solved the solution for me. I actually get the RGBW selector but it drive only the green color. I don't have the device anymore ( I switched for th fibaro one). 
I don't think that the origin of the RGBW missing is there, the Qubino uses 4 multilevel switches  and should receive also the COLOR information as stated in the manual. I have a very short knowledge of how Z-wave works so I took the easy solution by changing the device I use.

Thanks for having a look at it though !
